release/_publishing.sh

```diff
@@ -144,7 +144,7 @@ function upload_release {
 
 	for file in * .*
 	do
-		if [ -f "${file}" ]
+		if [ -f "${file}" ] && ! [[ "${file}" == *.jar* || "${file}" == *.pom* ]]
```

Comment:
@pgydev, shouldn't we also prevent the copy of the files with extension `MD5` and `sha512`?

Reply:
@natocesarrego  do you mean about the `.jar.MD5`, `.jar.sha512`, `.pom.MD5`, `.pom.sha512`,  or every `.MD5` and `sha512` because this will exclude  these ones `.jar.MD5`, `.jar.sha512`, `.pom.MD5`, `.pom.sha512`

Reply:
I meant to say `.jar.MD5, .jar.sha512, .pom.MD5, .pom.sha512` :slightly_smiling_face: